batch/batch/worker/worker.py

```diff
@@ -2044,6 +2053,21 @@ async def cleanup(self):
                 self.format_version, self.batch_id, self.job_id, self.attempt_id, 'main', log_contents
             )
 
+        if self.profile_file is not None:
+            with self.step('uploading_profile'):
+                if os.path.exists(self.profile_file):
+                    profile_contents = await self.worker.fs.read(self.profile_file)
+                else:
+                    profile_contents = b'profile not available'
+                await self.worker.file_store.write_jvm_profile(
+                    self.format_version,
+                    self.batch_id,
+                    self.job_id,
+                    self.attempt_id,
+                    'main',
+                    profile_contents,
+                )
```

Comment:
Writing 'profile not available' feels a little wrong to me. I feel like if there's a profile file in GCS I expect it to have valid profiling data in it. What message is this trying to convey? Should we be logging an error or warning in this case?